datafusion/common/src/file_options/parquet_writer.rs

```diff
@@ -141,6 +181,8 @@ impl TryFrom<&TableParquetOptions> for ParquetWriterOptions {
                     builder.set_column_max_statistics_size(path, max_statistics_size);
             }
         }
+
+        // ParquetWriterOptions will have defaults for the remaining fields (e.g. key_value_metadata & sorting_columns)
```

Comment:
This might be useful to include as a comment higher up in the doc comments as well, perhaps on `TableParquetOptions`